rts/ExternalAI/EngineOutHandler.cpp

```diff
@@ -348,7 +348,7 @@ void CEngineOutHandler::UnitDestroyed(const CUnit& destroyed, const CUnit* attac
 			if (attackerInLosOrRadar || hostSkirmishAIs[aiID].CheatEventsEnabled())
 				visibleAttackerId = attackerId;
 
-			hostSkirmishAIs[aiID].UnitDestroyed(destroyedId, visibleAttackerId);
+			hostSkirmishAIs[aiID].UnitDestroyed(destroyedId, visibleAttackerId, weaponDefID);
```

Comment:
AI interface will need some thinking about since a change will break compilation.

Reply:
I would assume AI compilation is part of the build I'm running. From the build logs:

```
[ 25%] Built target BARb
[ 34%] Built target CircuitAI
```